db/migrate/20180416210608_translations_tagline_drop_default.rb

```diff
@@ -0,0 +1,6 @@
+class TranslationsTaglineDropDefault < ActiveRecord::Migration[5.0]
+  def change
+    change_column_default(:translations, :translated_tagline, nil)
+    Translation.where( translated_tagline: '' ).update_all( translated_tagline: nil )
```

Comment:
`update_all` skips callbacks

Reply:
I'm not sure what this means.. can you explain?